cmd/main.go

```diff
@@ -395,3 +411,31 @@ func refreshCertPeriodically() {
 		}
 	}
 }
+
+// updatePodLabels updates the role label of pods based on the master election.
+func updatePodLabels(mgr manager.Manager) error {
+	c := mgr.GetClient()
+	// Fetch all pods in the given namespace
+	podList := &corev1.PodList{}
+	if err := c.List(context.TODO(), podList, &client.ListOptions{Namespace: nsxOperatorNamespace}); err != nil {
+		return fmt.Errorf("failed to list pods in namespace %s: %w", nsxOperatorNamespace, err)
+	}
+
+	// Iterate over the pods and update the role labels
+	for _, po := range podList.Items {
+		targetRole := roleStandby
+		if po.Name == nsxOperatorPodName {
+			targetRole = roleMaster
+		}
+
+		if po.Labels["role"] != targetRole {
```

Comment:
Can we only add label for master, no need to add standby label, right?

Reply:
I guess in that case, we need to remove old master label when there is a leader election again.

Reply:
yes, it will be removed.  I think `standby`  is helpful for debugging.